test/app/invitations_test.rb

```diff
@@ -89,6 +89,14 @@ def test_invitation
     refute team.includes?(charlie)
   end
 
+  def test_team_creator_receive_no_notification
```

Comment:
Lol the thumbs up on that isn't encouraging 😆 if you have any questions (or just want tell me I'm wrong), we can always jump on gitter or screenhero. I'm free to help.